src/ngSanitize/filter/linky.js

```diff
@@ -23,26 +34,14 @@
  * @example
    <example module="linkyExample" deps="angular-sanitize.js">
      <file name="index.html">
-       <script>
-         angular.module('linkyExample', ['ngSanitize'])
-           .controller('ExampleController', ['$scope', function($scope) {
-             $scope.snippet =
-               'Pretty text with some links:\n'+
-               'http://angularjs.org/,\n'+
-               'mailto:us@somewhere.org,\n'+
-               'another@somewhere.org,\n'+
-               'and one more: ftp://127.0.0.1/.';
-             $scope.snippetWithTarget = 'http://angularjs.org/';
-           }]);
-       </script>
        <div ng-controller="ExampleController">
        Snippet: <textarea ng-model="snippet" cols="60" rows="3"></textarea>
        <table>
-         <tr>
-           <td>Filter</td>
-           <td>Source</td>
-           <td>Rendered</td>
-         </tr>
+         <thead>
+           <th>Filter</th>
+           <th>Source</th>
+           <th>Rendered</th>
+         </thead>
```

Comment:
I think you are right that you shouldn't use `<thead>` without `<tr>`, even though browser generate the `<tr>`. What I really wanted was to have the `<th>` because that improves the formatting.